src/commands/PlayCommand.ts

```diff
@@ -76,6 +78,21 @@ export class PlayCommand extends BaseCommand {
                 return message.channel.send(createEmbed("error", `I could not load the playlist\nError: **\`${e.message}\`**`));
             }
         }
+
+        if (/^https?:\/\/(open\.spotify\.com)\/playlist(.*)$/g.exec(url)) {
+            try {
+                const tracks = await getTracks(url);
+                const playlistTitles: string[] = tracks.map((track: { artists: { name: string }[]; name: string }): string => `${track.artists[0].name} - ${track.name}`);
+                playlistTitles.map(async title => {
+                    const videoResults = await this.client.youtube.searchVideos(title, this.client.config.searchMaxResults);
+                    const queuedVideo = await this.client.youtube.getVideo(videoResults[0].id);
```

Comment:
'queuedVideo' is assigned a value but never used

Reply:
"dont review yet" :v

Reply:
:apasih: